### DISCUSSION TITLE
Multitarget to .NET 9?

### DISCUSSION THREAD

**Discussion by clin1234** [Devolutions/UniGetUI#2156] [General]
Just out of curiosity, would you be willing to include .NET 9 prerelease in your framework targets within the .csprojs. Also, what UI framework is used?

**Comment by marticliment** [comment]
.NET 9 is still on prerelease, so not yet.

the used framework is WinUI

**Comment by clin1234** [comment]
Now that .NET 9 is now stable, maybe reconsider?

**Reply by marticliment** [reply]
Ngl, I don't see the point of makign UniGetUI multirarget to .NET9, there are no important new features from which UniGetUI would benefit on NET9. Also, since this is not a library people are using with different .NET versions, I don't see any benefit from supporting NET8 and NET9. Furthermore, NET8 is LTS, while NET9 is STS, (.NET8 will be supported for longer than .NET9) so I reckon it makes more sense to stick to .NET8 and then jump to .NET10 directly, in the same way NET6 support was dropped in favour of .NET8.
